### DISCUSSION TITLE
新版本更新是否取消了一键匿名等相关功能

### DISCUSSION THREAD

**Discussion by Linriversluv** [note286/xduts#113] [Q&A] [answered]
大佬好。同学用\documentclass[anonrvw]{xdupgthesis}设置了盲审格式，不过这应该是旧版的命令吧。我看手册上并未提及这个，自己用着最新版在overleaf上试了一下也没有任何效果。另外，\anon命令只在正文里生效，那么info中关于姓名、学院等内容应该如何隐藏？

**Comment by note286** [comment, marked as answer]
请认真阅读手册！！！

参考 https://github.com/note286/xduts/issues/61 。

**Reply by Linriversluv** [reply]
了解，需要自行删除（笑哭.jpg）。那页眉能修改吗？

**Reply by note286** [reply]
选中对应的行，多行注释就可以了，很方便的。你要将页眉修改为啥？

**Reply by Linriversluv** [reply]
页眉不是写着西安电子科技大学XX学位论文嘛，这个在盲审格式里也是需要隐藏的吧？

**Reply by note286** [reply]
我不知道你学院是什么要求，如果有要求移除，手册中已经提及了可以移除页眉的，你Ctrl+F`页眉`。

**Reply by Linriversluv** [reply]
了解了，感谢答复！

**Reply by note286** [reply]
如果你不清楚，请咨询你的班长或者学院负责学位的老师，而不是自己想象。

**Reply by note286** [reply]
另外，多少来个star。

**Reply by Linriversluv** [reply]
（叼花.jpg）
